config.toml

```diff
@@ -1,4 +1,4 @@
-baseurl = "https://ukrainer-in-trier.github.io"
+baseurl = "https://www.ukrainer-in-trier.com"
```

Comment:
It looks weird because I already fixed it today. Changes are in develop.
https://github.com/Ukrainer-In-Trier/ukrainerintrier.github.io/blob/develop/config.toml